using-nats/nats-tools/nsc/signing_keys.md

```diff
@@ -224,9 +240,57 @@ Since the signing key has a unique role name within an account, it can be subseq
 ```shell
 nsc add user U -K service
 ```
+
 ```
 [ OK ] generated and stored user key "UBFRJ6RNBYJWSVFBS7O4ZW5MM6J3EPE75II3ULPVUWOUH7K7A23D3RQE"
 [ OK ] generated user creds file `~/test/issue-2621/keys/creds/synadia/A/U.creds`
 [ OK ] added user "U" to account "A"
 ```
-To see the permissions for the user enter `nsc describe user` - you will see in the report  that the user is scoped, and has the permissions listed. You can inspect and modify the scopedpermissions with `nsc edit signing keys` - pushing updates to the account will reassign user permissions
\ No newline at end of file
+
+To see the permissions for the user enter `nsc describe user` - you will see in the report  that the user is scoped, and has the permissions listed. You can inspect and modify the scopedpermissions with `nsc edit signing keys` - pushing updates to the account will reassign user permissions
+
+### Template functions
+
+*Available as of NATS 2.9.0*
+
+Although scoped signing are very useful and improve security, by limiting the scope of a particular signing key, the permissions that are set may be too rigid in multi-user setups. For example, given two users `pam` and `joe`, we may want to allow them to subscribe to their own namespaced subject in order to service requests, e.g. `pam.>` and `joe.>`. The permission _structure_ is the same between these users, but they differ in the concrete subjects which are further scoped to some property about that user.
```

Comment:
'Although scoped signing...' if this is singular it should be '...signing is' but if it's supposed to be plural we should change it to 'Although scoped signings are'

Reply:
Addressed. Missing `keys`